tez-dag/src/main/java/org/apache/tez/dag/app/RecoveryParser.java

```diff
@@ -659,11 +661,11 @@ public DAGRecoveryData parseRecoveryData() throws IOException {
     List<Path> summaryFiles = getSummaryFiles();
     LOG.debug("SummaryFile size:" + summaryFiles.size());
     for (Path summaryFile : summaryFiles) {
-      FileStatus summaryFileStatus = recoveryFS.getFileStatus(summaryFile);
-      LOG.info("Parsing summary file"
-          + ", path=" + summaryFile.toString()
-          + ", len=" + summaryFileStatus.getLen()
-          + ", lastModTime=" + summaryFileStatus.getModificationTime());
+      if (LOG.isDebugEnabled()) {
```

Comment:
I believe we might want to keep this on INFO level
recovery is not a heavily used code path under normal circumstances, and AN extra filesystem call due to getFileStatus is fine (especially if we're deep inside in debugging a non-reproducible recovery issue, where we usually want to see summary file info every time)

Reply:
reverted